bait-files/src/main/webapp/review.js

```diff
@@ -49,16 +49,13 @@ function getRequest() {
       .then(response => response.json())
       .then((request) => {
         addRequestToPage(request);
-
-        let blobKeyString = request.image;
-        if (blobKeyString != 'noKey') {
-          fetch('/blobstore-get-image?blobKey=' + blobKeyString).then((pic) => {
-            // append picture element to page
-            let picture = document.createElement('img');
-            picture.src = pic.url;
-            document.getElementById(PICTURE_INPUT).append(picture);
-          });
+        addImageToPage(request);
+        if (request.redirect) {
+          alert('RequestId invalid. Redirecting to request portal.');
+          window.location.replace('/requests.html');
+          return;
         }
+        addRequestToPage(request);
```

Comment:
This doesn't look right? addRequestToPage is called twice?

Reply:
removed one of the calls, must've been from the pull from master